settings.json

```diff
@@ -0,0 +1 @@
+{"editor.formatOnSave": true}
```

Comment:
@mike This file is used by vscode. I think it should be in gitignore for now

Reply:
Added to git ignore